bootstrap/sql/migrations/native/1.5.0/mysql/schemaChanges.sql

```diff
@@ -75,3 +75,7 @@ CREATE TABLE IF NOT EXISTS api_endpoint_entity (
     UNIQUE (fqnHash),
     INDEX (name)
 );
+
+-- Remove date, dateTime, time from type_entity, as they are no more om-field-types, instead we have date-cp, time-cp, dateTime-cp as om-field-types
+DELETE FROM type_entity
```

Comment:
@sonika-shah what happens if there is an installation using these types?

Reply:
hi @harshach  , om-field-types have been used only for custom properties, earlier in prev versions we do not had custom property enabled for date,dateTime and time , so it will be safe to remove the om-field-type from these old fields and have them in new fields-date-cp, dateTime-cp,time-cp